src/coreclr/vm/stackwalk.cpp

```diff
@@ -554,7 +554,11 @@ PCODE Thread::VirtualUnwindCallFrame(T_CONTEXT* pContext,
     PT_RUNTIME_FUNCTION pFunctionEntry;
 
 #if !defined(TARGET_UNIX) && defined(TARGET_ARM64)
-    if ((pContext->ContextFlags & CONTEXT_UNWOUND_TO_CALL) != 0)
+    // We don't adjust the control PC when we have a code info, as the code info is always created from an unadjusted one
```

Comment:
Is the unadjusted code info going to work well for the unwind? In other words, can we hit the bug that you have fixed when the code info is passed in?

Reply:
The code info is passed in only for managed frames. If it was not the case, the code info would not be valid and the assert I am fixing would be firing. 
Return address in managed code is never out of the bounds of the method containing the call, otherwise unwinding managed code would be crashing / hanging on arm64 Windows both with and without the new EH.